iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/user/user.component.html

```diff
@@ -66,7 +66,14 @@ <h1>
 
                         <!-- Send reset password email or change password-->
                         <user-password user="$ctrl.user"></user-password>
-
+                        <button ng-if="$ctrl.user.aupSignature"
```

Comment:
We'd prefer to move this logic in a sub-component (like reset password button one line above)

Reply:
Sub-component created.